python/xorbits/datasets/iterable_dataset.py

```diff
@@ -0,0 +1,530 @@
+# Copyright 2022-2023 XProbe Inc.
+# derived from copyright 1999-2021 Alibaba Group Holding Ltd.
```

Comment:
Remove this line in the new created files. Same as other places.

Reply:
All the files in datasets contains this line, I will create a new PR to remove them.